frontend/web/src/app/components/home/home.component.html

```diff
@@ -1,3 +1,9 @@
-<p>
-  home works!
-</p>
+<div class="container" id="wrapper">
+  <div class="logo">
+      <img src="http://resources.mynewsdesk.com/image/upload/t_open_graph_image/y2pmtfolvlimvq6ajvkv.jpg" />
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AImgWithoutAltCheck)